tests/test_downloads.py

```diff
@@ -143,8 +143,8 @@ def test_download_cached(self, export, get_cached):
         assert not export.delay.called
 
     def test_download_forbidden(self):
-        with pytest.raises(ApiError):
-            self.client.post_json(api.url_for(resource.DownloadView, path='elections', office='house', cycle=2018))
+        response = self.app.get(api.url_for(resource.DownloadView, path='elections', office='house', cycle=2018))
```

Comment:
I am unclear about what this test is trying to accomplish.

Reply:
not clear either, Can we remove this test case function? Thanks @vrajmohan

Reply:
Is there another view that _does not support_ downloads that can be used for this test instead?

Reply:
good question. I look most of our endpoints, we do have some endpoints don't support download, but they are not added into this test_download function. 
How about change function name to `donttest_download_forbidden` for now?  in case, we need test other endpoints in the future? Thanks

Reply:
I don't understand "they are not added into this test_download_forbidden function". Why can't we use one of those _paths_ instead?

Reply:
good idea, I add '/elections/search' (resource.ElectionListView) endpoint into test_download_forbidden function. work well. Thanks.